packages/control/src/control.ts

```diff
@@ -80,6 +80,15 @@ export abstract class Control<T extends HTMLElement = HTMLElement> {
     this._node.addEventListener(event, callback);
   }
 
+  public once(event: string, callback: (e: Event) => void) {
```

Comment:
`addEventListener()` can recieve the options object that contains `once` property. If this property is set to true, then listener automaticly removes after its called

https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#once

I think you should rework this method or remove it altogether